sources/missing.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "Missing",
 	"name": "missing",
 	"prefix": "xx",
-    	"key":"",
+	"key": "",
 	"url": "",
-	"license": "",
-        "description": "Missing - as in a placeholder for WOF records without a geometry"
-}
+	"license": "N/A",
+	"license_type": "",
+	"license_text": "",
+	"description": "Missing - as in a placeholder for WOF records without a geometry."
```

Comment:
Hmm... I'm hesitant to remove this since it is currently on the `missing.json` source file. Both this source and the `unknown.json` file should have a description, but since they are essentially placeholders for "we don't know," I'm not entirely sure if there is a better description here.